### DISCUSSION TITLE
Warn before deleting Saved replies

### DISCUSSION THREAD

**Discussion by notator** [community/community#22142] [New to GitHub]
<p>I’m using Saved replies (accessed from my settings). When I hit a reply’s “delete” button, the reply gets deleted without warning, so I have to be <em>very</em> careful not to click it by accident when aiming for the “edit” button.  Could you build in a “warning” message?<br>
Thanks,<br>
notator</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi there notator! :wave: Welcome to the Community!</p>
<p>Thanks for this feedback! We’re always working to improve GitHub and the GitHub Support Community, and we consider every suggestion we receive.</p>
<p>Would you mind <a href="https://support.github.com/contact/feedback">submitting this through our official product feedback form</a> so that our product team can track your request?</p>

**Comment by notator** [comment]
<aside class="quote no-group" data-username="notator" data-post="1" data-topic="118630" data-full="true">
<div class="title">
<div class="quote-controls"></div>
 notator:</div>
<blockquote>
<p>I’m using Saved replies (accessed from my settings). When I hit a reply’s “delete” button, the reply gets deleted without warning, so I have to be <em>very</em> careful not to click it by accident when aiming for the “edit” button. Could you build in a “warning” message?<br>
Thanks,<br>
notator</p>
</blockquote>
</aside>
<p>Done!<br>
All the best,<br>
notator</p>
